location/__init__.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import json
+import re
+import requests
+
+sys.path.append("../")
+from utils import db, ms
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+if os.environ.get("GOOGLE_MAPS_KEY"):
+    key = os.environ.get("GOOGLE_MAPS_KEY")
+    REQUEST_URL = f"https://maps.googleapis.com/maps/api/geocode/json?key={key}"
+    REQUEST_URL += "&address={}"
```

Comment:
why do these need to be in separate statements?

Reply:
The empty {} isn't allowed by the f-string

Reply:
what about with `.format()`?